apps/backoffice-v2/src/domains/auth/hooks/queries/useAuthenticatedUserQuery/useAuthenticatedUserQuery.tsx

```diff
@@ -2,5 +2,9 @@ import { useQuery } from '@tanstack/react-query';
 import { authQueryKeys } from '../../../query-keys';
 
 export const useAuthenticatedUserQuery = () => {
-  return useQuery(authQueryKeys.authenticatedUser());
+  return useQuery({
+    ...authQueryKeys.authenticatedUser(),
+    staleTime: 100_000,
+    refetchInterval: 100_000,
```

Comment:
I made these changes to a few queries, explanation:
1. `staleTime` - Prevent the query from refetching on every new query observer.
2. `refetchInterval` - Override the default 3-second polling. Not every query should be polled every 3 seconds.